default/en/introduction.md

```diff
@@ -17,15 +17,15 @@ The middle sections (5-9) contain the guidelines for RISM's authority files. The
 Visit [RISM's Muscat page](https://rism.info/community/muscat.html) for more information about Muscat.
 
 The guidelines are maintained by the RISM Editorial Center in Frankfurt, Germany.  
-English: Jennifer Ward (jennifer.ward@rism.info)
-German: Martina Falletta (martina.falletta@rism.info)
-Should anything be unclear, do not hesitate to contact us.
-
-We would also like to extend our gratitude to our translators, whose efforts make Muscat more easily accessible to our colleagues worldwide.
-Catalan: Ferran Jorba
-Italian: Claudio Bacciagaluppi, Rodolfo Zitellini
-French: Claudio Bacciagaluppi, Rodolfo Zitellini, Laurent Pugin
-Korean: JeongYoun Chang 
-Polish: Marcelina Chojecka, Katarzyna Diehl
-Portuguese: André Guerra Cotta (†), Diósnio Machado Neto, Beatriz Magalhães Castro, Silvia Sequeira, Marcos Virmond
-Spanish: Rodrigo Balaguer, Antonio Ezquerro, John Lazos, Lucas Reccitelli, Oliver Schöner, Luciana Giron Sheridan
+English: [Jennifer Ward](mailto:jennifer.ward@rism.info)  
+German: [Martina Falletta](mailto:martina.falletta@rism.info)  
+Should anything be unclear, do not hesitate to contact us.  
+
+We would also like to extend our gratitude to our translators, past and present, whose efforts make Muscat more easily accessible to our colleagues worldwide.  
+Catalan: Ferran Jorba  
```

Comment:
This section should probably be formatted as a list?

Reply:
Oh I thought it was, but I forgot the dashes. Will do that on the next pass. 
All suggestions gratefully accepted!